src/app/public/plugin-resources/code-examples/split-view/basic/split-view-demo.component.ts

```diff
@@ -0,0 +1,164 @@
+import {
+  Component
+} from '@angular/core';
+
+import {
+  FormControl,
+  FormGroup
+} from '@angular/forms';
+
+import {
+  Subject
+} from 'rxjs';
+
+import {
+  SkyConfirmCloseEventArgs,
+  SkyConfirmService,
+  SkyConfirmType
+} from '@skyux/modals';
+
+import {
+  SkySplitViewMessage,
+  SkySplitViewMessageType
+} from '@skyux/split-view';
+
+@Component({
+  selector: 'sky-split-view-demo',
+  templateUrl: './split-view-demo.component.html'
+})
+export class SkySplitViewDemoComponent {
```

Comment:
Demo type names should be ambiguous:
```suggestion
export class SplitViewDemoComponent {
```

Reply:
Done